lib/src/sep/0099/hotswap.dart

```diff
@@ -6,23 +6,24 @@ import 'dart:async';
 import 'dart:convert';
 
 import 'package:http/http.dart' as http;
+import 'package:stellar_flutter_sdk/src/extensions/extensions.dart';
 import 'package:stellar_flutter_sdk/stellar_flutter_sdk.dart';
 
 class Hotswap {
-  late String _hotswapServerAddress;
+  late final String _serverAddress;
   late http.Client httpClient;
 
   Hotswap(
-    this._hotswapServerAddress, {
+    String serverAddress, {
     http.Client? httpClient,
   }) {
-    this.httpClient = httpClient == null ? http.Client() : httpClient;
```

Comment:
If we create it we also need a dispose for the http client we created manually.

Reply:
Done.